modules/examples/eb-sf-batch/images/simulation-mock/src/simulation_mock/sqs_manager.py

```diff
@@ -0,0 +1,142 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License").
+#    You may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import argparse
+import os
+import sys
+import time
+from contextlib import contextmanager
+from datetime import timedelta
+from typing import Iterator
+
+from platonic.queue.errors import MessageReceiveTimeout
+from platonic.sqs.queue import SQSReceiver
+from platonic.sqs.queue.message import SQSMessage
+from platonic.sqs.queue.types import ValueType
+from platonic.timeout import ConstantTimeout
+from simulation_mock import get_logger
+
+MESSAGE_RECEIVE_TIMEOUT_SECONDS = 90
+PROCESSING_TIMEOUT_MINUTES = 5
+FILE_CHECK_SLEEP_TIME_SECONDS = 30
+DATA_FILE_NAME = "message.json"
+
+LOGGER = get_logger()
+
+
+class ProcessingTimeoutException(Exception):
+    pass
+
+
+class SQSHeartbeatReceiver(SQSReceiver[ValueType]):
```

Comment:
Is this used by the code or is it a leftover from the simulation example? Could we directly reference the application code for the simulation example instead of having a copy?

Reply:
We could also have a super simple example batch application instead of the simulation if it reduces confusion

Reply:
I am not using the simulation code here, i need to cleanup. I am using a simple shell command to make sure my job ran successfully or not `Line100 of stack.py`